### PR TITLE
Add non-generated gRPC classes under checkstyle/findbugs coverage

### PR DESCRIPTION
**Pull request by ggezer**
Earlier maven exclusions for generated classes were covering gRPC helper/manager classes as well since they use the same package. This PR narrows exclusions down to core-transport module and fixes hidden checkstyle/findbugs issues.

### CODE REVIEW

**Comment by ggezer** [comment]
@gpang PTAL.

File: build/findbugs/findbugs-exclude.xml [hunk 1]
```diff
@@ -2,13 +2,7 @@
 
   <Match>
     <Or>
-      <Package name="org.apache.thrift"/>
```
Comment (gpang) [inline build/findbugs/findbugs-exclude.xml]: we don't actually have this package now, right?

File: core/common/src/main/java/alluxio/grpc/ServiceVersionClientServiceHandler.java [hunk 2]
```diff
@@ -31,11 +32,16 @@
   /** Set of services that are going to be recognized by this versioning service. */
   private Set<ServiceType> mServices;
 
+  /**
+   * Creates service version handler that allows given services.
+   * @param services services to allow
```
Comment (gpang) [inline core/common/src/main/java/alluxio/grpc/ServiceVersionClientServiceHandler.java]: should there be a blank line before this?

File: build/findbugs/findbugs-exclude.xml (hunk 1 above)
Comment (ggezer) [reply build/findbugs/findbugs-exclude.xml]: We don't

File: core/common/src/main/java/alluxio/grpc/ServiceVersionClientServiceHandler.java (hunk 2 above)
Comment (ggezer) [reply core/common/src/main/java/alluxio/grpc/ServiceVersionClientServiceHandler.java]: You mean a space between general doc and the @param doc? I can but it's pretty common through out the code.

**Comment by ggezer** [comment]
@gpang No code change. PTAL.

**Comment by gpang** [comment]
alluxio-bot, merge this please

**Pull request merged**